pdudaemon/drivers/energenieusb.py

```diff
@@ -103,9 +103,8 @@ def accepts(cls, drivername):
     # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
     def connect(self):
-        """
-        Returns the list of compatible devices.
-        @return: device list
+        """Returns the list of compatible devices.
+        @return: device list.
```

Comment:
`@return: device list.` is not a sentence, so adding `.` doesn't seem to make much sense. Why is this needed?

Reply:
This seems to be the auto-fix for the [rule related to ending in a punctuation](https://docs.astral.sh/ruff/rules/ends-in-punctuation/)

Because there is no new line between the first line of the docstring it assumed the two lines are a single sentence, hence it adds the full stop at the end.

It would be possible to have the following instead:

```suggestion

        @return: device list
```

Reply:
I've updated to add whitespace instead.